utils/basicTippingContractUtil.js

```diff
@@ -22,7 +22,7 @@ const formatSingleClaim = (contractId, url, [claimGen, amount]) => {
 const formatClaims = returnState => {
   const state = returnState.decodedResult;
 
-  return state.claims ? state.claims.map(([url_id, rawClaim]) => {
+  return state.claims && state.urls ? state.claims.map(([url_id, rawClaim]) => {
```

Comment:
is this change related to V4?

Reply:
I noticed it as issue, but I think it generally could have happened